messaging/src/main/java/org/axonframework/eventhandling/scheduling/dbscheduler/DbSchedulerEventScheduler.java

```diff
@@ -398,6 +412,18 @@ public Builder startScheduler(boolean startScheduler) {
             return this;
         }
 
+        /**
+         * Sets whether to stop the {@link Scheduler} using the {@link Lifecycle}, or to never stop the scheduler from
+         * this component instead. Defaults to {@code true}.
+         *
+         * @param stopScheduler a {@code boolean} to determine whether to stop the scheduler.
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder stopScheduler(boolean stopScheduler) {
```

Comment:
Same method rename suggestion as on the `DbSchedulerDeadlineManager` :-)

Reply:
IDK, it's now the same as the `startScheduler`. And consistent with, for example, `extendedGapCheckEnabled` on the JdbcEventStorageEngine. I can also imagine users want to explicitly set it to true.

Reply:
That makes total sense. Let's go for the consistency and stick to a similar approach as the `startScheduler` method!